components/bl-static-breadcrumbs/src/index.js

```diff
@@ -0,0 +1,26 @@
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function StaticBreadcrumbsComponent({ component, eventHandlers, elRef }) {
+  const { classList, display, style, options } = component;
+  const { onItemClick, onMouseOver, onMouseOut } = eventHandlers;
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div ref={ elRef } className={ cn('bl-customComponent-staticBreadcrumbs', classList) } style={ style }>
+      { options?.map(item => (
```

Comment:
How about to add options validator with readable errors that will tell user the signature of page item?

Also it helps to remove optional chaining here
P.S: for me question marks always looks dirty:)